examples/bela/bela_example_digitalout.scd

```diff
@@ -1,16 +1,12 @@
-Server.default = s = Server("belaServer", NetAddr("192.168.7.2", 57110));
-s.initTree;
-s.startAliveThread;
+Server.default = Server.remote("belaServer", NetAddr("192.168.7.2", 57110));
```

Comment:
could these be more compact like 

```
s =  Server.remote("belaServer", NetAddr("192.168.7.2", 57110));
``` 

? (I don't actually know the answer)

Reply:
it would work, but it's conceptually not so good in my opinion. That would assign a remote server to `s`, but won't make it the default server: e.g. code that uses Server.default would still access the local server. I don't have examples from the top of my head, but I would prefer to keep Server.default there.

Reply:
ok thanks